packages/prettier-plugin-liquid/src/printer/print/liquid.ts

```diff
@@ -490,6 +496,45 @@ export function printLiquidRawTag(
   return [blockStart, ...body, blockEnd];
 }
 
+export function printLiquidDoc(
+  path: AstPath<RawMarkup>,
+  _options: LiquidParserOptions,
+  print: LiquidPrinter,
+  _args: LiquidPrinterArgs,
+) {
+  const body = path.map((p: any) => print(p), 'nodes');
+  return [indent([hardline, join(hardline, body)]), hardline];
+}
+
+export function printLiquidDocParam(
+  path: AstPath<LiquidDocParamNode>,
+  _options: LiquidParserOptions,
+  _print: LiquidPrinter,
+  _args: LiquidPrinterArgs,
+): Doc {
+  const node = path.getValue();
+  const parts: Doc[] = ['@param'];
+
+  if (node.paramType.value) {
+    parts.push(' ', `{${node.paramType.value}}`);
+  }
+
+  if (node.paramName.value) {
+    parts.push(' ', node.paramName.value);
+  }
+
+  if (node.paramDescription.value) {
+    const normalizedDescription = node.paramDescription.value.replace(/\s+/g, ' ').trim();
+    if (node.paramDescription.dashSeparated) {
+      parts.push(' - ', normalizedDescription);
+    } else {
+      parts.push(' ', normalizedDescription);
+    }
```

Comment:
IMO this is usually the place where it should be _configurable_ and homogenized by prettier. Forgot to put the dash? We'll put it for you. The goal is to have a consistent result, _not_ to repeat what the input had.

Reply:
https://github.com/Shopify/theme-tools/pull/646/commits/f028514e5196b62db1eb0dbfec068cc0e568699b